src/app/components/views/job-view/job-view.component.ts

```diff
@@ -74,6 +76,22 @@ export class JobViewComponent implements OnInit {
     }
   }
 
+  /**
+   * @description - Subscribe to job service in order to get currently selected job data, and then verify that the given job exists. To be called in ngOnInit()
+   * @returns {void}
+   */
+  private subscribeToJobService(): void {
+    this.jobService.getLoading().pipe(takeUntil(this.ngUnsubscribe)).subscribe({
+      next: (loading: boolean) => {
+        this.jobApiLoading = loading;
+        if (!this.jobApiLoading) {
+          this.job = this.jobService.getSelectedJob();
```

Comment:
can/should we just unsubscribe in here?

Reply:
should this even be a subscription

Reply:
it is a subscription in every job subview component.

Reply:
I went ahead and removed the subscription from every child view component for the job. Like the job, customer, history, etc tabs no longer subscribe. They just grab the job data from the service. There's no way they'd grab it unless it was there because: 

a) you can only access a job page if you have a job to click. You can only have a job to click if the job has been retrieved from the api.

b) if you refresh the page within a job, the job header bar thing will route back to the homepage if it sees no job is selected (meaning you can't view an "empty" job)